Packs/Cisco-umbrella-cloud-security/Integrations/CiscoUmbrellaCloudSecurityv2/CiscoUmbrellaCloudSecurityv2.yml

```
@@ -0,0 +1,490 @@
+    name: umbrella-add-domain
+  - arguments:
+    - description: Optional orgId, by default uses the one set in the instance configuration
+      name: orgId
+    - description: Destination list ID to get domains from. Use umbrella-get-destination-lists to get the list ID
+      name: destId
+      required: true
+    deprecated: true
+    description: Gets the domains listed in a destination list
+    name: umbrella-get-destination-domains
+    outputs:
+    - contextPath: Umbrella.Destinations.id
+      description: ID of domain within destination list
+    - contextPath: Umbrella.Destinations.destination
+      description: Domain within destination list
+    - contextPath: Umbrella.Destinations.type
+      description: Type of destination within destination list
+    - contextPath: Umbrella.Destinations.createdAt
+      description: When the domain within destination list was created
+    - contextPath: Umbrella.Destinations.comment
+      description: Comment associated with domain within destination list
+  - arguments:
+    - description: Optional organization ID. If not provided, will use the one provided in the integration configuration.
+      name: orgId
+    - description: Destination list ID
+      name: destId
+      required: true
+    - description: 'List of entry IDs to remove from destination list (Format: 1234,1235)'
+      isArray: true
+      name: domainIds
+      required: true
+    deprecated: true
+    description: Removes domains from given destination list
+    name: umbrella-remove-domain
+  - arguments:
+    - description: Optional orgId, by default uses the one set in the instance configuration
+      name: orgId
+    - description: Destination list ID to get domains from. Use umbrella-get-destination-lists to get the list ID
+      name: destId
+      required: true
+    - description: Domain to get from a destination list
+      name: domain
+      required: true
+    deprecated: true
+    description: Gets the domain from a destination list
+    name: umbrella-get-destination-domain
+    outputs:
+    - contextPath: Umbrella.Destinations.id
+      description: ID of domain within destination list
+    - contextPath: Umbrella.Destinations.destination
+      description: Domain within destination list
+    - contextPath: Umbrella.Destinations.type
+      description: Type of destination within destination list
+    - contextPath: Umbrella.Destinations.createdAt
+      description: When the domain within destination list was created
+    - contextPath: Umbrella.Destinations.comment
+      description: Comment associated with domain within destination list
+  - arguments:
+    - description: Optional orgId, by default uses the one set in the instance configuration
+      name: orgId
+    - description: Destination list ID to get domains from. Use umbrella-get-destination-lists to get the list ID
+      name: destId
+      required: true
+    - description: Domains to search for in a destination list
```

```suggestion
    - description: Domains to search for in a destination list.
```
Should this be: A comma-separated list of domains to search for in a destination list.
(I see that isArray is true.)